### ISSUE TITLE
Name property is not defined for “Inequalities” button in graphing mode

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1127]
**Repro Steps:** 
1. Launch 'Calculator' and turn on narrator. 
2. Home screen will be displayed.​
3. Navigate through Menu using keyboard and select 'Graphing'. 'Graphing' Calculator opens.​
4. Navigate to "Inequalities" combo box.
5. Observe the issue.

**Actual Result:**
Name property is not defined for “Inequalities” button.

**Expected Result:**
Name property should be defined for “Inequalities” button.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
